src/Umbraco.Infrastructure/Examine/ContentValueSetBuilder.cs

```diff
@@ -72,7 +75,7 @@ private IEnumerable<ValueSet> GetValueSetsEnumerable(IContent[] content, Diction
             {
                 { "icon", c.ContentType.Icon?.Yield() ?? Enumerable.Empty<string>() },
                 {
-                    UmbracoExamineFieldNames.PublishedFieldName, new object[] { c.Published ? "y" : "n" }
+                    UmbracoExamineFieldNames.PublishedFieldName, c.Published ? NoValue : YesValue
```

Comment:
@Shazwazza why are the values reversed here?

I guess it should be:

```
UmbracoExamineFieldNames.PublishedFieldName, c.Published ? YesValue : NoValue
```

Reply:
oops my mistake, i'll update